vignettes/quickstart.Rmd

```diff
@@ -88,54 +67,60 @@ function include:
 
 - `data` the primary longitudinal data.frame containing the outcome variable and all covariates
 - `data_ice` a data.frame specifying which visit (if any) the patient's intercurrent 
-event occurred on. If the patient had multiple intercurrent events (ICE) this should 
-specify the date on which the first non-MAR intercurrent event occurred on. It also 
+event (ICE) occurred on, or more precisely the first visit in which the outcome has been affected by the ICE. If the patient had multiple ICEs this should 
+specify the first visit affected by the ICE addressed with a non-MAR imputation strategy. It also 
 specifies which reference based imputation strategy we want to use.
 - `method` specifies what method we want to use to fit our imputation models as well as what
 method we want to use to generate our imputed values. 
 
-For the purpose of this vignette we shall say that the patients ICE visit is the
-first visit in which a missing value has occurred and that all patients will be
-imputed under the "MAR" strategy. We will create 150 imputation models using bayesian
+In our example the patients ICE visit is the
+first visit in which a missing value has occurred. We assume that all patients will be
+imputed under the Jump To Reference (JR) strategy. We will create 150 imputation models using Bayesian
 methods to sample the model coefficients from their posterior distributions for a model 
-of `outcome ~ 1 + age + sex + group * visit`.
+of `change ~ 1 + basval * VISIT + THERAPY * VISIT`.
 
 ```{r}
-# create data_ice setting the imputation method to MAR for
+# create data_ice setting the imputation method to JR for
 # each patient with at least one missing value
 dat_ice <- dat %>% 
-    arrange(id, visit) %>% 
-    filter(is.na(outcome)) %>% 
-    group_by(id) %>% 
+    arrange(PATIENT, VISIT) %>% 
+    filter(is.na(change)) %>% 
+    group_by(PATIENT) %>% 
     slice(1) %>%
     ungroup() %>% 
-    select(id, visit) %>% 
-    mutate(strategy = "MAR")
+    select(PATIENT, VISIT) %>% 
+    mutate(strategy = "JR")
+
+# The patient with id 3618 is the unique one that has an intermittent missing values.
+# Actually he does not stop the treatment -> remove from data_ice
```

Comment:
I'm not really sure exactly what this comment is trying to say

Reply:
in the previous line of code I set the visit_ICE corresponding to the first NA I observe in the outcome of a patient. But I have to remove this patient from data_ice since he does not experience the ICE, he just has a missing value. 

```
# The patient with id 3618 is the unique one that has an intermittent missing values -> remove from data_ice since he
# does not experience any ICE.
```